libs/compute_api/src/spec.rs

```diff
@@ -145,6 +146,10 @@ pub struct ComputeSpec {
     /// over the same replication content from publisher.
     #[serde(default)] // Default false
     pub drop_subscriptions_before_start: bool,
+
+    /// Set of JSON web keys that the compute can use to authenticate
+    /// communication from the control plane.
+    pub jwks: Option<JwkSet>,
```

Comment:
Maybe it's implicitly handled but what happens when an empty compute sends get spec request to the cplane. Do we fill `jwks` and use it to start compute_ctl API correctly?

Reply:
See the changes in https://github.com/neondatabase/neon/pull/10530/commits/e817bf308d96b331438bb38accb8b70b53bf2f6a. Thanks for pointing me at the relevant cplane code.